v6/jbuild

```diff
@@ -15,7 +15,7 @@ let flags = function
     in
     go ic ""
 
-let rewriters_ppx = ["ppx_deriving_rpc"; "ppx_sexp_conv"]
+let rewriters_ppx = ["ppx_deriving_rpc" ]
```

Comment:
I wasn't really sure about removing `ppx_sexp_conv`, but it's not used in the squeezed jbuild, and I was able to locally build all v6 executables without any issue after removing this. Please let me know if it needs to be included.

Reply:
It's fine, thanks for the cleanup